src/theme/palette.ts

```diff
@@ -34,32 +23,69 @@ const GREY_TONES = {
   500: "#404040",
   700: "#252525",
 }
-
 const lightPalette: PaletteOptions | undefined = {
   mode: "light",
   primary: {
-    main: saddleColors.seafoam,
+    main: "#06D7D7",
```

Comment:
Why did we remove `saddleColors`?

Reply:
After giving color on the theme we will use these color over all the theme. So, actually we only need this color on the theme file. And it would be not convenient to give name on saddle color , also on the editor, after installing extension, we can see color on the hex or rgb color.
So, I removed saddleColor.

Also if we need another color name it would be good to give on the palette theme after declare types. Then we can access to this color from anywhere of our app.